Mathlib/Data/Finset/Image.lean

```diff
@@ -408,22 +408,24 @@ theorem fiber_nonempty_iff_mem_image (f : α → β) (s : Finset α) (y : β) :
 #align finset.fiber_nonempty_iff_mem_image Finset.fiber_nonempty_iff_mem_image
 
 @[simp, norm_cast]
-theorem coe_image {f : α → β} : ↑(s.image f) = f '' ↑s :=
+theorem coe_image : ↑(s.image f) = f '' ↑s :=
   Set.ext <| by simp only [mem_coe, mem_image, Set.mem_image, implies_true]
 #align finset.coe_image Finset.coe_image
 
+@[simp]
+lemma nonempty_image_iff : (s.image f).Nonempty ↔ s.Nonempty := by
```

Comment:
While you're at it, the finset naming convention would rather choose
```suggestion
lemma image_nonempty : (s.image f).Nonempty ↔ s.Nonempty := by
```

Reply:
I aligned the name with `Set.nonempty_image_iff`.

Reply:
Yes, but `Finset` and `Set` differ in naming. FWIW I am of the opinion that it should be `Set.image_nonempty` too.

Reply:
OK, let's try this.